### ISSUE TITLE
Pareto Stacked bar looses dimension when plotted Q vs Q

### ISSUE THREAD

**Issue by fabioannovazzi** [fabioannovazzi/mparanza_bugs#19]
The file is

![image](https://user-images.githubusercontent.com/3429553/140944657-e2948137-74c9-42f4-af50-b86488d7b578.png)

All fine with this setting

![image](https://user-images.githubusercontent.com/3429553/140944768-d1b3bab3-739c-4682-b718-f3fc50a1d81a.png)

and these 5 dimensions are available in the pareto stacked bar chart

![image](https://user-images.githubusercontent.com/3429553/140944968-3a87c94e-df6e-4e43-b05e-f0f98ee7aad0.png)

returning this 

![image](https://user-images.githubusercontent.com/3429553/140945099-da44d114-d2e1-4a5e-88ae-0392742f59d9.png)

When set so

![image](https://user-images.githubusercontent.com/3429553/140945245-b49d9582-1dbb-4c09-bf91-78aac3c57e19.png)

Paese disappears from the options

![image](https://user-images.githubusercontent.com/3429553/140945361-c8da5853-306e-4eca-b760-070cdc576b3b.png)

Basically in one case there is Paese and not Articolo and in the other case the other way round.

**Comment by fabioannovazzi** [comment]
What happened is that, to optimize performance in the variable dimension variance calculation, the app dropped the two different columns in the two different time settings. And the columns were not available to plot the mekko chart. The dropping was done also when calculating one dimension variance

![image](https://user-images.githubusercontent.com/3429553/141085584-b1284ec7-cace-4247-bcf9-8544d0dfcc7e.png)

Now if one dimension variance is chosen it does not drop anything so the columns are the same for time dimensions. 

![image](https://user-images.githubusercontent.com/3429553/141085885-ce51026e-0234-461a-a5af-52fe05cc6d94.png)
